src/signals_notebook/entities/samples/cell.py

```diff
@@ -73,9 +75,16 @@ class SampleCellBody(BaseModel):
 
 
 class SampleCell(BaseModel):
-    id: Optional[Union[UUID, str]]
+    id: Optional[str]
     name: Optional[str]
     content: SampleCellContent = Field(default=SampleCellContent())
+    read_only: Optional[bool] = False
+
+    def _is_cell_mutable(self):
+        if self.name == 'Amount':
```

Comment:
Is this solution the best for detecting mutable Cells?

Reply:
ohh, yes